randovania/layout/preset_migration.py

```diff
@@ -718,6 +718,16 @@ def _migrate_v42(preset: dict) -> dict:
         preset = _update_default_dock_rando(preset)
     return preset
 
+def _migrate_v43(preset: dict) -> dict:
+    game = default_database.game_description_for(RandovaniaGame(preset["game"]))
```

Comment:
It's best to not use the DB here, as it can change in the future and break the migration of old presets.

Reply:
Am I allowed to use `migration_data.json` here, too? I already bloated the json to find the start node name for the description migration.

Reply:
Yeah, the purpose of that json is for schema migrations like this

Reply:
Done. Did a small refactor so that both migrations can use the same function